examples/ct/ctmapper/mapper/mapper.go

```diff
@@ -22,17 +22,18 @@ import (
 	"time"
 
 	"github.com/golang/glog"
-	pb "github.com/golang/protobuf/proto"
 	"github.com/golang/protobuf/ptypes"
 	"github.com/golang/protobuf/ptypes/any"
-	ct "github.com/google/certificate-transparency-go"
 	"github.com/google/certificate-transparency-go/client"
 	"github.com/google/certificate-transparency-go/jsonclient"
 	"github.com/google/certificate-transparency-go/x509"
 	"github.com/google/trillian"
 	"github.com/google/trillian/examples/ct/ctmapper"
 	"github.com/google/trillian/examples/ct/ctmapper/ctmapperpb"
 	"google.golang.org/grpc"
+
+	pb "github.com/golang/protobuf/proto"
+	ct "github.com/google/certificate-transparency-go"
```

Comment:
This is on me, I've been doing that for a while now. I must have passed a few changes like that by you, but maybe you didn't notice?

@gdbelvin, I don't want to block your PR with a sidelined discussion on import style, so I'm happy either way here.

Reply:
I thought the convention was to put renamed and underscore imports at the bottom?

Reply:
It's not a big deal, so don't hold up the PR on going one way or the other.

I've got no problem with files being laid out that way (as @codingllama does), and I don't really object to requiring that files be laid out that way (as this PR will presumably enforce) --  but there are ~30 existing files (in c-t-go) which don't do this, so it's definitely a new convention.

(Aside: the main culprit in c-t-go is the top-level package, which has a package name of `ct` but which has `certificate-transparency-go` as the last path component.  Technically, it could be imported as-is (`import "github.com/google/certificate-transparency-go"`) but that's confusing both for humans and simpler tools)

Reply:
This PR won't enforce any particular ordering for imports. 
I just made the changes while I was adjusting other imports.  Any thoughts on whether that's a habit to continue or discontinue?

Reply:
Ah, OK, I assumed that the new tool was going to enforce this -- if not, no objections at all.

Reply:
no enforcement :-) I don't know of any linter that does that.